### PR TITLE
Test update1.13.0

### PR DESCRIPTION
**Pull request by crmorford**
Adding the features we've written since 1.9.0 to the overlaying tests.

### CODE REVIEW

File: plottable.js [hunk 1]
```diff
@@ -908,7 +908,7 @@ var Plottable;
 ///<reference path="../reference.ts" />
 var Plottable;
 (function (Plottable) {
-    Plottable.version = "1.13.0";
+    Plottable.version = "1.12.0";
```
Comment (bluong) [inline plottable.js]: Oh no...

File: quicktests/overlaying/overlaying.css [hunk 2]
```diff
@@ -170,3 +170,13 @@ ol{
...
+
+.plottable .guide-line-layer.red line.guide-line {
+    stroke: #F00;
+    stroke-width: 1px;
+}
+
+.plottable .guide-line-layer.green line.guide-line {
+    stroke: #0F0;
+    stroke-width: 1px;
+}
```
Comment (bluong) [inline quicktests/overlaying/overlaying.css]: Add newline EOF please.

File: quicktests/overlaying/tests/animations/inflation.js [hunk 3]
```diff
@@ -1,94 +0,0 @@
...
-
-  for( var i = 0; i < plots.length; i++){
-    var plot = plots[i];
-    addClick(plot);
-  }
-
-  var lineChart = new Plottable.Components.Table([[yAxis, group],
-                                                 [null,  xAxis]]);
-  lineChart.renderTo(svg);
-}
```
Comment (bluong) [inline quicktests/overlaying/tests/animations/inflation.js]: Just verifying.  Do you want this test to go away?

**Comment by bluong** [comment]
Bunch of eslint issues.

File: quicktests/overlaying/tests/animations/inflation.js (hunk 3 above)
Comment (crmorford) [inline quicktests/overlaying/tests/animations/inflation.js]: yep, it's been redone with segment plots!

**Comment by crmorford** [comment]
favorite locale: https://jsfiddle.net/zsfkg32b/17/

**Pull request merged**